package.json

```diff
@@ -38,13 +39,12 @@
     "lodash.isequal": "^4.5.0",
     "lodash.uniqby": "^4.7.0",
     "lodash.uniqueid": "^4.0.1",
-    "md-editor-rt": "^4.7.0",
     "markdown-it-abbr": "^1.0.4",
     "markdown-it-container": "^3.0.0",
     "markdown-it-deflist": "^2.1.0",
     "markdown-it-emoji": "^2.0.2",
     "markdown-it-footnote": "^3.0.3",
-    "markdown-it-html5-embed": "^1.0.0",
+    "markdown-it-html5-embed": "^0.3.3",
```

Comment:
why it was downgraded?

Reply:
because new version 1.0.0 has a vulnerability. when it will be fixed and will be released a new version, when we can upgrade it